config/login.dfe.support.dev.json

```diff
@@ -26,55 +42,151 @@
     "type":"redis",
     "params": {
-      "connectionString": "redis://localhost:6379"
+      "connectionString": "redis://:BUeq3HkJzVvPRTLeNhl8MgLrFF+gLUkgvboHrwLMCF4=@s141d01-signin-shd-rc.redis.cache.windows.net:6380/3?tls=true"
     }
   },
   "directories": {
-    "type": "static"
+    "type":"api",
+    "service": {
+      "url": "https://s141d01-signin-dir.azurewebsites.net",
+      "auth": {
+        "type": "aad",
+        "tenant": "platformidentity.onmicrosoft.com",
+        "authorityHostUrl": "https://login.microsoftonline.com/9c7d9dd3-840c-4b3f-818e-552865082e16",
+        "clientId": "350e8e5d-3875-44da-878d-1bb7feb7125d",
+        "clientSecret": "Fj=owAV=m9@477wzLVgNPo]VT?4YZUnW",
+        "resource": "b220f0ce-ad01-49c2-aa72-379aec0cb096"
+      }
+    }
   },
   "organisations": {
-    "type": "static"
-  },
-  "devices": {
-    "type": "static"
+    "type":"api",
+    "service": {
+      "url": "https://s141d01-signin-org.azurewebsites.net/",
+      "auth": {
+        "type": "aad",
+        "tenant": "platformidentity.onmicrosoft.com",
+        "authorityHostUrl": "https://login.microsoftonline.com/9c7d9dd3-840c-4b3f-818e-552865082e16",
+        "clientId": "350e8e5d-3875-44da-878d-1bb7feb7125d",
+        "clientSecret": "Fj=owAV=m9@477wzLVgNPo]VT?4YZUnW",
+        "resource": "b220f0ce-ad01-49c2-aa72-379aec0cb096"
+      }
+    }
   },
   "audit": {
-    "type": "redis",
+    "type": "sequelize",
     "params": {
-      "connectionString": "redis://localhost:6379"
+      "host" :"s141d01-signin-shd-sql.database.windows.net",
+      "username":"3Z4R66fCgTfaXbkukgpP",
+      "password":"KGUTHYfWpUC2!amns7DF",
+      "dialect":"mssql",
+      "name":"s141d01-signin-audit-db",
+      "encrypt": true,
+      "schema": "dbo",
+      "pool": {
+        "max": 100,
+        "min": 0,
+        "acquire": 30000,
+        "idle": 10000
+      }
     },
-    "cacheConnectionString": "redis://localhost:6379"
+    "cacheConnectionString": "redis://:BUeq3HkJzVvPRTLeNhl8MgLrFF+gLUkgvboHrwLMCF4=@s141d01-signin-shd-rc.redis.cache.windows.net:6380/7?tls=true"
   },
   "serviceMapping": {
     "type": "redis",
     "params": {
-      "connectionString": "redis://localhost:6379"
+      "connectionString": "redis://:BUeq3HkJzVvPRTLeNhl8MgLrFF+gLUkgvboHrwLMCF4=@s141d01-signin-shd-rc.redis.cache.windows.net:6380/3?tls=true"
     },
-    "key2SuccessServiceId": ""
+    "key2SuccessServiceId": "3BFDE961-F061-4786-B618-618DEAF96E44"
+  },
+  "devices": {
+    "type": "api",
+    "service": {
+      "url": "https://s141d01-signin-dev.azurewebsites.net",
+      "auth": {
+        "type": "aad",
+        "tenant": "platformidentity.onmicrosoft.com",
+        "authorityHostUrl": "https://login.microsoftonline.com/9c7d9dd3-840c-4b3f-818e-552865082e16",
+        "clientId": "350e8e5d-3875-44da-878d-1bb7feb7125d",
+        "clientSecret": "Fj=owAV=m9@477wzLVgNPo]VT?4YZUnW",
+        "resource": "b220f0ce-ad01-49c2-aa72-379aec0cb096"
+      }
```

Comment:
I'd like to clarify that this configuration should be present on the integration branch.